src/main/java/seedu/address/logic/autocomplete/CommandParameterSyntaxHandler.java

```diff
@@ -0,0 +1,117 @@
+package seedu.address.logic.autocomplete;
+
+import static seedu.address.logic.parser.CliSyntax.PREFIX_COST;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_DATE_END;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_DATE_START;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_EMAIL;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_ITEM_NUMBER;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_NAME;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_NEW_ROOM;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_PASSWORD;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_PHONE;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_ROOM;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_TAG;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_USERNAME;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+
+import seedu.address.logic.commands.AddCommand;
+import seedu.address.logic.commands.CheckInCommand;
+import seedu.address.logic.commands.CheckoutCommand;
+import seedu.address.logic.commands.LogInCommand;
+import seedu.address.logic.commands.ReassignCommand;
+import seedu.address.logic.commands.ServiceCommand;
+import seedu.address.logic.parser.Prefix;
+
+/**
+ * Contains command Syntax definitions for multiple commands
+ */
+public class CommandParameterSyntaxHandler {
+    public static final ArrayList<Prefix> ADD_COMMAND_PREFIXES = getListOfPrefix(PREFIX_NAME,
```

Comment:
I get what you're trying to say but how do I extract the prefixes from the AddCommand class?
Also, I did it this way because it allows me to freely change the order of prefixes.